File: CS50/pset6/mario/more/mario.py
height = int(input("Height: "))

while height < 1 or height > 8:
    print("Please enter a value between 1 to 8.")
    height = int(input("Height: "))
    
z = height

for i in range(0, height):
    for k in range(z - 1, 0, -1):
        print(" ", end="")
    print("#", end="")
    for y in range(i):
        print("#", end="")
    z-= 1
    print("  ",end="")
    for q in range(i + 1):
        print("#", end="")
    print()

File: CS50/pset1/mario/more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    printf("Enter a Height: \n");

    int n= get_int(0);

    if(n < 1 || n > 8)
    {
    do
    {
        printf("Please enter a value between 1 to 8.\n");
        n= get_int(0);
        if (n >= 1 && n <= 8)
        {
            break;
        }
    } while (n < 1 || n > 8);
    }

    int z = n;

    for(int i = 0; i < n; i++) //loop for rows
    {
        for(int k = z - 1; k > 0; k--) //loop for spaces in nth row
        {
        printf(" ");
        }
        printf("#");

        for(int y = 0; y < i; y++)
        {
            printf("#");
        }
        z--;

        printf("  ");

        for(int q = 0; q <= i; q++)
        {
            printf("#");
        }
        printf("\n");
    }
}

